<!-- -->
<launch>
  <arg name='host' default='localhost'/>
  <arg name='port' default='2000'/>
  <arg name='timeout' default='10'/>
  <arg name="role_name" default="ego_vehicle" />
  <arg name="track" default="t1_triple" />

  <param name="/carla/host" value="$(arg host)" />
  <param name="/carla/port" value="$(arg port)" />
  <param name="/carla/timeout" value="$(arg timeout)" />

  <node pkg="waypoint_node" type="waypoint_node.py" name="waypoint_node_$(arg role_name)" output="screen">
    <param name="role_name" value="$(arg role_name)" />
    <param name="track" value="$(arg track)" />
  </node>

</launch>
